azure-pipelines.yml

```
@@ -22,11 +22,11 @@ jobs:
     vmImage: 'ubuntu-latest'
   steps:
   - script: |
-      echo "FROM quay.io/pypa/manylinux1_x86_64
+      echo "FROM quay.io/pypa/manylinux1_686
```

The image is i686 (missing the `i`). I would suggest using manylinux2014 rather than manylinux1
```suggestion
      echo "FROM quay.io/pypa/manylinux2014_i686
```